samples/Chat/src/app/ConfigurationScreen.tsx

```diff
@@ -116,6 +124,43 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
     internalSetupAndJoinChatThread();
   }, [name, joinChatHandler, selectedAvatar, setDisplayName, setEndpointUrl, setThreadId, setToken, setUserId]);
 
+  const joinChatThreadWithExistingUser = useCallback(() => {
+    const internalSetupAndJoinChatThread = async (): Promise<void> => {
```

Comment:
why do we have a wrapper on this? should this just be the function itself? does this also need to be async?